client/src/app/router.coffee

```diff
@@ -0,0 +1,11 @@
+Marionette = require('backbone.marionette')
+Controller = require('./controller')
+
+module.exports = Marionette.AppRouter.extend(
+  controller: new Controller()
+  appRoutes:
+    'new_site': 'new_site'
+    'list_sites': 'list_sites'
```

Comment:
lets change these routes to just `/sites` and `/sites/:id`  instead of `list_sites`

Also, `/sites/new` for a new site instead of `new_site`

This will allow us to follow the more "normal" RESTful pattern of naming